makefile

```
@@ -143,7 +150,12 @@ cdk-synth:
 
 cdk-diff:
 	$(call log, "Showing CDK diff...")
-	cd cdk && pnpm run diff
+	@if [ -z "$$AWS_PROFILE" ]; then \
+		echo "Error: AWS_PROFILE is not set - please run 'export AWS_PROFILE=<profile>' before running this command."; \
+    	exit 1; \
+	else \
+		cd cdk && pnpm run diff; \
```

Could we set the AWS_PROFILE here for the user as I take it this project is only deployed to the one AWS account?


```suggestion
		cd cdk && AWS_PROFILE=frontend pnpm run diff; \
```